patches/tModLoader/Terraria/Player.TML.cs

```diff
@@ -85,6 +85,8 @@ public bool MeleeEnchantActive {
 	/// </summary>
 	public const int ManaCrystalMax = 9;
 
+	public StatModifier breathEffectiveness = StatModifier.Default;
+
```

Comment:
Needs docs. Explain that an effectiveness of 2 means breath lasts twice as long. Recommend modders use additive? 

Is there something without the Base/Flat?

Reply:
Vanilla is all multiplicative, but that can get out of hand pretty quickly I imagine. I'm not seeing a class like StatModifier without base/flat. We have AddableFloat and MultipliableFloat classes.